packages/material-ui/src/AccordionSummary/AccordionSummary.d.ts

```diff
@@ -1,6 +1,6 @@
 import * as React from 'react';
 import { SxProps } from '@material-ui/system';
-import { ExtendButtonBase, ExtendButtonBaseTypeMap } from '../ButtonBase';
+import { ExtendButtonBase, ExtendButtonBaseTypeMap } from '@material-ui/core/ButtonBase';
```

Comment:
```suggestion
import { ExtendButtonBase, ExtendButtonBaseTypeMap } from '../ButtonBase';
```

Reply:
I agree, public API imports from same package don't make sense. Tests should test public API 👍

Reply:
> Tests should test public API 👍

I'll just focus on that. Thank you all.

Reply:
Revert ready, I will advance with the other tests later